src/qibolab/channel/channel_map.py

```diff
@@ -2,22 +2,9 @@
 
 from qibo.config import raise_error
 
-from .channel_configs import ChannelConfig
+from . import Channel
 
-
-@dataclass(frozen=True)
-class Channel:
-    """Channel is an abstract concept that defines means of communication
-    between users and a quantum computer.
-
-    A quantum computer can be perceived as just a set of channels where
-    signals can be sent to or received from. Channels are identified
-    with a unique name. The type of a channel is inferred from the type
-    of config it accepts.
-    """
-
-    name: str
-    config: ChannelConfig
+__all__ = ["ChannelMap"]
```

Comment:
I support the usage of `__all__`, it will allow us to properly define the public API.